src/visitor.ts

```diff
@@ -20,10 +20,9 @@ export type VaryOptions = {
 };
 
 export type AbOptions = {
-  callback: (assignment: boolean | string) => void;
+  callback: (assignment: boolean) => void;
   context: string;
-  trueVariant: boolean | string;
-  visitor: Visitor;
+  trueVariant?: string;
```

Comment:
Why not `trueVariant?: boolean | string;`?

Reply:
That works too! I don't feel strongly about this. I do think (and this is a contrived example) that it would make it possible for someone to pass `trueVariant: false` and have the client think that they meant `true` because of how this is implemented though.

Reply:
ah, yeah, essentially the `false || 'true'` bit?

Reply:
I buy that keeping this stringy forces people to use a non-falsey way of specifying false.